AGM_Medical/clientInit.sqf

```diff
@@ -56,6 +56,7 @@ AGM_Pain_CC ppEffectCommit 0;
       AGM_Blinding_CC ppEffectEnable False;
       AGM_Pain_CC ppEffectEnable False;
       AGM_Pain_CA ppEffectEnable False;
+      AGM_player setBleedingRemaining 0;
```

Comment:
clientInit is executed once per PC. This variable should most likely be set on Unit init, not client init.